cdap-api-spark/src/main/scala/co/cask/cdap/api/spark/SparkMain.scala

```diff
@@ -162,12 +201,12 @@ trait SparkMain extends Serializable {
     /**
       * Creates a [[org.apache.spark.rdd.RDD]] from the given [[co.cask.cdap.api.dataset.Dataset]].
       *
-      * @param datasetName name of the Dataset
-      * @param arguments arguments for the Dataset; default is an empty [[scala.collection.Map]]
+      * @param datasetName name of the dataset
```

Comment:
Why changing all the cases? I thought we also use `Dataset` to refer to the CDAP dataset?